docs/CONFIGURATION.md

```diff
@@ -124,31 +124,41 @@ is not specified then processor execution order will be random.
-* **fieldpass**: An array of strings that is used to filter metrics generated by the
-current input. Each string in the array is tested as a glob match against field names
-and if it matches, the field is emitted. fieldpass is not available for outputs.
-* **fielddrop**: The inverse of pass, if a field name matches, it is not emitted.
-fielddrop is not available for outputs.
-* **tagpass**: tag names and arrays of strings that are used to filter
-measurements by the current input. Each string in the array is tested as a glob
-match against the tag name, and if it matches the measurement is emitted.
-* **tagdrop**: The inverse of tagpass. If a tag matches, the measurement is not
-emitted. This is tested on measurements that have passed the tagpass test.
-* **tagexclude**: tagexclude can be used to exclude a tag from measurement(s).
-As opposed to tagdrop, which will drop an entire measurement based on it's
-tags, tagexclude simply strips the given tag keys from the measurement. This
-can be used on inputs & outputs, but it is _recommended_ to be used on inputs,
-as it is more efficient to filter out tags at the ingestion point.
-* **taginclude**: taginclude is the inverse of tagexclude. It will only include
-the tag keys in the final measurement.
-
-**NOTE** `tagpass` and `tagdrop` parameters must be defined at the _end_ of
-the plugin definition, otherwise subsequent plugin config options will be
-interpreted as part of the tagpass/tagdrop map.
+* **namepass**:
+An array of strings that is used to filter metrics generated by the
+current input. Each string is tested as a glob match against measurement names
+and only if it matches, the field is emitted.
+* **namedrop**: The inverse of `namepass`.  If the measurement name matches it is
+discarded.
+* **fieldpass**:
+An array of strings that is used to filter metrics generated by the current
+input. Each string in the array is matched as a glob pattern against field
+names and if it matches, the field is emitted. `fieldpass` is not available
+for outputs.
+* **fielddrop**:
+The inverse of `fieldpass`, if a field key matches, it is not emitted.
+`fielddrop` is not available for outputs.
+* **tagpass**:
+A table mapping tag keys to arrays of tag value patterns.  Each pattern is
+matched as a glob against the corresponding tag key and if the point contains
+a matching tag value then the point is emitted.
+* **tagdrop**:
+The inverse of `tagpass`.  If a match is found, the point is not emitted. This
+is tested on points after they have passed the `tagpass` test.
+* **taginclude**:
+Used to whitelist tags on a point.  In contrast to `tagpass`, which will
+pass an entire point based on its tag, `taginclude` passes only the specified
+tags from the measurement.  This filter can be used on both inputs & outputs,
+but it is _recommended_ to be used on inputs, as it is more efficient to
+filter out tags at the ingestion point.
```

Comment:
The description doesn't cover how to format the config param. An array of tag names to glob. non-matching tag is dropped.